hazelcast-sql/src/test/java/com/hazelcast/sql/SqlBasicTest.java

```diff
@@ -547,11 +548,17 @@ public static SerializationConfig serializationConfig() {
     }
 
     static Config memberConfig() {
-        Config config = new Config().setSerializationConfig(serializationConfig());
-
-        config
+        Config config = new Config()
```

Comment:
Why this test requires such extensive setup now?

Reply:
Good notice, thanks.
Actually, my first priority was to just make them work with Jet backend. Optimization was planned afterwards :)